stdlib/public/stubs/LibcShims.cpp

```diff
@@ -21,14 +26,25 @@
 #include <bsd/stdlib.h>
 #endif
 
+#if defined(_MSC_VER)
+static_assert(std::is_same<long long, swift::__swift_ssize_t>::value,
```

Comment:
This assertion is correct as is.  Please fix the definition of `__swift_ssize_t` instead.